src/data/v2/last_quote.rs

```diff
@@ -173,12 +199,24 @@ mod tests {
     let api_info = ApiInfo::from_env().unwrap();
     let client = Client::new(api_info);
 
-    let req = LastQuoteReqInit::default().init("SPY");
-    let quote = client.issue::<Get>(&req).await.unwrap();
+    let req = LastQuoteReq::new("SPY");
+    let quotes = client.issue::<Get>(&req).await.unwrap();
     // Just as a rough sanity check, we require that the reported time
     // is some time after two weeks before today. That should safely
     // account for any combination of holidays, weekends, etc.
-    assert!(quote.time >= Utc::now() - Duration::weeks(2));
+    assert!(quotes[0].time >= Utc::now() - Duration::weeks(2));
+  }
+
+  /// Retrieve multiple symbols at once.
+  #[test(tokio::test)]
+  async fn request_last_quotes_multi() {
+    let api_info = ApiInfo::from_env().unwrap();
+    let client = Client::new(api_info);
+
+    let req = LastQuoteReq::new("SPY,QQQ,MSFT");
```

Comment:
I don't think we should be passing in some kind of pre-formatted string. It should likely be a vec/iter/slice/whatever of strings. If I recall correctly there is precedent in other parts of the API surface that we should follow, unless not suitable for some reason.

Reply:
it's what alpaca wants, but I agree it's a bit icky. will fix